crates/dojo/world/src/local/artifact_to_local.rs

```diff
@@ -14,7 +14,7 @@ use super::*;
 use crate::config::ProfileConfig;
 
 const WORLD_INTF: &str = "dojo::world::iworld::IWorld";
-const CONTRACT_INTF: &str = "dojo::contract::interface::IContract";
```

Comment:
If we don't use `IContract`/`IModel`/`IEvent`, in this case we can use as you've done the `ContractImpl`.

So:
1. We check we have the `DEPLOYED_RESOURCE_INTF` as the type: `dojo::meta::IDeployedResource;`.
2. Then, we use the implementation name with determined convention and we're good.